doc/getting_started/getting_started.rst

```diff
@@ -70,7 +70,7 @@ To build an example application follow these steps:
 
```

Comment:
while you're in there, how about amending the previous line to read (for clarity):

     #. Navigate to the main project directory (where you cloned the Zephyr source from GitHub):

Reply:
Done.